cli/cmd/test_run_cmd.go

```diff
@@ -31,7 +31,7 @@ var testRunCmd = &cobra.Command{
 		environmentOptions := append(baseOptions, actions.WithClient(utils.GetResourceAPIClient("environments", cliConfig)))
 		environmentActions := actions.NewEnvironmentsActions(environmentOptions...)
 
-		runTestAction := actions.NewRunTestAction(cliConfig, cliLogger, client, environmentActions)
+		runTestAction := actions.NewRunTestAction(cliConfig, cliLogger, client, environmentActions, ExitCLI)
```

Comment:
Related with this comment: https://github.com/kubeshop/tracetest/pull/2617/files#r1209657040